pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -427,6 +457,24 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		domain.Spec.Devices.Watchdog = newWatchdog
 	}
 
+	if vmi.Spec.Domain.Devices.Rng != nil {
```

Comment:
so how do I explicitely disable virtio-rng if I do not want to see it in VM?

Reply:
@michalskrivanek I remember we had the discussion about it, what the reason that you do not want rng device at all?

Reply:
@cynepco3hahue we did indeed, and AFAICT the outcome was that we want it by default but opting-out is important as well. There are OSes/arches where this device doesn't work/doesn't make sense

Reply:
@michalskrivanek Works for me, thanks for clarification.